src/metax_api/api/rpc/base/views/common_rpc.py

```diff
@@ -25,10 +23,6 @@ class CommonRPC(CommonViewSet):
 
     api_type = 'rpc'
 
-    # serves no purpose, but satisfies the ViewSet basic requirements
-    serializer_class = FileSerializer
-    queryset = File.objects.all()
```

Comment:
Why delete these and add serializer_class and queryset to FileRPC and StatisticRPC separately?

Reply:
DatasetRPC is actually using serializer in recently added APIs, but otherwise will fix this!